public/components/ViewTypeToggle.vue

```diff
@@ -58,6 +60,7 @@ import {
   LATITUDE_TYPE,
   GEOCOORDINATE_TYPE,
   REMOTE_SENSING_TYPE,
+  GEOBOUNDS_TYPE,
```

Comment:
You forgot to include the updated `util.types.js` file in your PR.

Reply:
The GEOBOUNDS_TYPE already existed in the master.